projects/pipelines/toolbox/pipelines.py

```diff
@@ -39,6 +39,7 @@ def run_kfp_notebook(self,
                          stop_on_exit=True,
                          capture_artifacts=True,
                          capture_prom_db=False,
+                         user_count=0,
```

Comment:
default for `capture_extra_artifacts` should be `true`. Assume low-scale by default.

Reply:
Yeah that makes sense. I think having the threshold at all misses the point of the problem we were having anyway. I think I like the `capture_extra_artifacts` flag, since there is really only very specific situations where it should be turned off anyway. I'll switch that for both roles.